### PR TITLE
Improve schema

### PR DESCRIPTION
**Pull request by ignasposka**

### CODE REVIEW

**Review by AzisK — APPROVED** [review]
Looks good. Could you create a ticket to resolve user organization?

File: database/3-fake-data.sql [hunk 1]
```diff
@@ -15,7 +15,8 @@ INSERT INTO app_user
 ('aiubfaw4io09', 'Hi', 'Hello', 'Letena', 'hi@hi.lt'),
 ('afhu9w4f78', 'Carl', 'Red', 'Gauras', 'hi@hi.lt'),
 ('278y2378ryb', 'Bob', 'Blue', 'Eyes', 'hi@hi.lt'),
-('0932hfdsa', 'Mamba', 'Green', 'Jungleen', 'hi@hello.lt');
+('0932hfdsa', 'Mamba', 'Green', 'Jungleen', 'hi@hello.lt'),
+('userIdForTesting', 'Azis', 'Ąžuolas', 'Krušna', 'hi@hello.lt');
```
[the remark this answers is not in the document]
Comment (AzisK) [reply database/3-fake-data.sql:19]: Also, should we allow duplication of emails?

[the remark this answers is not in the document]
Comment (ignasposka) [reply database/3-fake-data.sql:19]: Looks like we do. Need to add an UNIQUE constraint to it.

**Comment by ignasposka** [comment]
> Looks good. Could you create a ticket to resolve user organization?

Sure

**Pull request merged**